tests/basic.rs

```diff
@@ -162,6 +162,51 @@ fn current_kv_dropped() {
     assert_eq!(Arc::strong_count(&dropped2), 1);
 }
 
+#[test]
+fn empty_maps_equal() {
+    let map1 = FlurryHashMap::<usize, usize>::new();
+    let map2 = FlurryHashMap::<usize, usize>::new();
+    assert_eq!(map1, map2);
+}
+
+#[test]
+fn different_size_maps_not_equal() {
+    let map1 = FlurryHashMap::<usize, usize>::new();
+    let map2 = FlurryHashMap::<usize, usize>::new();
+    {
+        let guard = epoch::pin();
+        map1.insert(1, 0, &guard);
+    }
+
+    assert_ne!(map1, map2);
+}
+
+#[test]
+fn same_values_equal() {
+    let map1 = FlurryHashMap::<usize, usize>::new();
+    let map2 = FlurryHashMap::<usize, usize>::new();
+    {
+        let guard = epoch::pin();
+        map1.insert(1, 0, &guard);
+        map2.insert(1, 0, &guard);
+    }
+
+    assert_eq!(map1, map2);
+}
+
+#[test]
+fn different_values_not_equal() {
+    let map1 = FlurryHashMap::<usize, usize>::new();
+    let map2 = FlurryHashMap::<usize, usize>::new();
+    {
+        let guard = epoch::pin();
+        map1.insert(1, 0, &guard);
+        map2.insert(1, 1, &guard);
+    }
+
+    assert_ne!(map1, map2);
+}
+
```

Comment:
We should add a test where the two maps are of different sizes.

We should also have all of these test for equality both ways:
```rust
assert_eq!(map1, map2);
assert_eq!(map2, map1);
```